src/main/cpp/t06_longest.cpp

```diff
@@ -17,10 +17,36 @@
 
 #include "t06_longest.h"
 #include <iostream>
+#include <string>
 
-
-using namespace std;
-
+std::string blabla(std::string line){
+    int curr_len = 0;
+    int max_len = 0;
+    std::string max_word = "";
+    std::string curr_word = "";
+    for(int i =0; i < line.length(); i++){
+        if (line[i] != ' '){
+            curr_len++;
+            curr_word += line[i];
+        }
+        else{
+            if(curr_len > max_len){
+                max_len = curr_len;
+                max_word = curr_word;
+            }
+            curr_len = 0;
+            curr_word = "";
+        }
+    }
+    if(curr_len > max_len){
+                max_len = curr_len;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'max_len' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4718331)